src/templates/drizzle/labelheader.hbs

```diff
@@ -0,0 +1,24 @@
+<{{tag}} {{#if id}}id="{{id}}"{{/if}} class="
+  drizzle-u-inlineBlock
+  drizzle-u-alignMiddle
+  drizzle-u-textNoWrap
+  drizzle-u-padRight">
+  {{#if key}}
```

Comment:
Good catch, @erikjung. Just to be clear, there are still 3 possible outcomes.

1. `<a href="./{{key}}.html">{{> @partial-block}}</a>`
2. `<a href="#{{id}}" class="drizzle-u-linkHidden">{{> @partial-block}}</a>`
3. `{{> @partial-block}}`

I'm not seeing an obvious way to more cleanly handle the differences between the first and the second.